src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -141,6 +144,28 @@ public static Set<Tag> parseTags(Collection<String> tags) throws ParseException
         return tagSet;
     }
 
+    /**
+     * Parses {@code String date} into an {@code LocalDate} and returns it. Leading and trailing whitespaces will be
+     * trimmed.
+     */
+    public static LocalDate parseDate(String date) throws ParseException {
```

Comment:
do you want to add in some exception checking for the input arguments? then maybe can throw the parseException if the formatter is not of the right pattern